perception/autoware_compare_map_segmentation/src/voxel_grid_map_loader/voxel_grid_map_loader.hpp

```diff
@@ -270,6 +271,10 @@ class VoxelGridDynamicMapLoader : public VoxelGridMapLoader
   {
     map_grid_size_x_ = map_cell_to_add.metadata.max_x - map_cell_to_add.metadata.min_x;
     map_grid_size_y_ = map_cell_to_add.metadata.max_y - map_cell_to_add.metadata.min_y;
+    if (map_grid_size_x_ > max_map_grid_size_ || map_grid_size_y_ > max_map_grid_size_) {
+      throw std::runtime_error(
```

Comment:
Please use `RCLCPP_ERROR` to handle ROS2 error like other nodes.

Reply:
Should this error invoke with dynamic map loading mode?

Reply:
@YoshiRi 
> Should this error invoke with dynamic map loading mode?

Yes, Since function `addMapCellAndFilter` is member of `VoxelGridDynamicMapLoade` Class so this error invoked during Dynamic map loader mode.

Reply:
> Please use `RCLCPP_ERROR` to handle ROS2 error like other nodes.

I fixed at https://github.com/autowarefoundation/autoware.universe/pull/9024/commits/1416bfbe79b8275c573459b38c42e09b72f42b62